2021-03-15 07:16:01,157 INFO    MainThread:25294 [wandb_setup.py:_flush():69] setting env: {'entity': 'saish', 'project': 'demo', 'sweep_id': 'gn0pd3id', 'root_dir': '/cbr/saish/PhD/DL/Assignment_1', 'run_id': 'xqm4qtuz', 'sweep_param_path': '/cbr/saish/PhD/DL/Assignment_1/wandb/sweep-gn0pd3id/config-xqm4qtuz.yaml'}
2021-03-15 07:16:01,157 INFO    MainThread:25294 [wandb_setup.py:_flush():69] setting login settings: {}
2021-03-15 07:16:01,158 INFO    MainThread:25294 [wandb_init.py:_log_setup():334] Logging user logs to /cbr/saish/PhD/DL/Assignment_1/wandb/run-20210315_071601-xqm4qtuz/logs/debug.log
2021-03-15 07:16:01,158 INFO    MainThread:25294 [wandb_init.py:_log_setup():335] Logging internal logs to /cbr/saish/PhD/DL/Assignment_1/wandb/run-20210315_071601-xqm4qtuz/logs/debug-internal.log
2021-03-15 07:16:01,158 INFO    MainThread:25294 [wandb_init.py:init():367] calling init triggers
2021-03-15 07:16:01,158 INFO    MainThread:25294 [wandb_init.py:init():374] wandb.init called with sweep_config: {'activation': 'tanh', 'batch_size': 16, 'epochs': 10, 'hlayer_size': 64, 'initializer': 'random', 'l_rate': 0.001, 'loss': 'squared_error', 'n_hlayers': 4, 'optimizer': 'sgd'}
config: {'batch_size': 64, 'l_rate': 0.001, 'optimizer': 'mgd', 'epochs': 5, 'activation': 'tanh', 'initializer': 'xavier', 'loss': 'cross_entropy'}
2021-03-15 07:16:01,159 INFO    MainThread:25294 [wandb_init.py:init():416] starting backend
2021-03-15 07:16:01,159 INFO    MainThread:25294 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-03-15 07:16:01,168 INFO    MainThread:25294 [backend.py:ensure_launched():123] starting backend process...
2021-03-15 07:16:01,172 INFO    MainThread:25294 [backend.py:ensure_launched():128] started backend process with pid: 25365
2021-03-15 07:16:01,173 INFO    MainThread:25294 [wandb_init.py:init():421] backend started and connected
2021-03-15 07:16:01,174 INFO    MainThread:25294 [wandb_run.py:_config_callback():733] config_cb None None {'activation': 'tanh', 'batch_size': 16, 'epochs': 10, 'hlayer_size': 64, 'initializer': 'random', 'l_rate': 0.001, 'loss': 'squared_error', 'n_hlayers': 4, 'optimizer': 'sgd'}
2021-03-15 07:16:01,174 INFO    MainThread:25294 [wandb_init.py:init():461] updated telemetry
2021-03-15 07:16:01,175 INFO    MainThread:25294 [wandb_init.py:init():480] communicating current version
2021-03-15 07:16:02,403 INFO    MainThread:25294 [wandb_init.py:init():485] got version response 
2021-03-15 07:16:02,403 INFO    MainThread:25294 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-03-15 07:16:02,703 INFO    MainThread:25294 [wandb_init.py:init():518] starting run threads in backend
2021-03-15 07:16:04,919 INFO    MainThread:25294 [wandb_run.py:_console_start():1480] atexit reg
2021-03-15 07:16:04,920 INFO    MainThread:25294 [wandb_run.py:_redirect():1343] redirect: SettingsConsole.REDIRECT
2021-03-15 07:16:04,920 INFO    MainThread:25294 [wandb_run.py:_redirect():1348] Redirecting console.
2021-03-15 07:16:04,920 INFO    MainThread:25294 [redirect.py:install():213] install start
2021-03-15 07:16:04,921 INFO    MainThread:25294 [redirect.py:install():228] install stop
2021-03-15 07:16:04,921 INFO    MainThread:25294 [redirect.py:install():213] install start
2021-03-15 07:16:04,921 INFO    MainThread:25294 [redirect.py:install():228] install stop
2021-03-15 07:16:04,921 INFO    MainThread:25294 [wandb_run.py:_redirect():1394] Redirects installed.
2021-03-15 07:16:04,921 INFO    MainThread:25294 [wandb_init.py:init():542] run started, returning control to user process
2021-03-15 07:28:14,532 INFO    MainThread:25294 [wandb_run.py:_config_callback():733] config_cb ('_wandb', 'viz', 'confusion_matrix') {'id': 'wandb/confusion_matrix/v1', 'historyFieldSettings': {'key': 'confusion_matrix', 'x-axis': '_step'}} None
2021-03-15 07:28:14,549 INFO    MainThread:25294 [wandb_run.py:_atexit_cleanup():1450] got exitcode: 0
2021-03-15 07:28:14,550 INFO    MainThread:25294 [wandb_run.py:_restore():1422] restore
2021-03-15 07:28:14,550 INFO    MainThread:25294 [redirect.py:uninstall():232] uninstall start
2021-03-15 07:28:14,550 INFO    MainThread:25294 [redirect.py:_stop():287] _stop: stdout
2021-03-15 07:28:14,551 INFO    stdout    :25294 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-03-15 07:28:14,551 INFO    MainThread:25294 [redirect.py:_stop():293] _stop closed: stdout
2021-03-15 07:28:14,551 INFO    stdout    :25294 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-03-15 07:28:14,551 INFO    MainThread:25294 [redirect.py:_stop():299] _stop joined: stdout
2021-03-15 07:28:14,552 INFO    MainThread:25294 [redirect.py:_stop():301] _stop rd closed: stdout
2021-03-15 07:28:14,552 INFO    MainThread:25294 [redirect.py:uninstall():236] uninstall done
2021-03-15 07:28:14,552 INFO    MainThread:25294 [redirect.py:uninstall():232] uninstall start
2021-03-15 07:28:14,552 INFO    MainThread:25294 [redirect.py:_stop():287] _stop: stderr
2021-03-15 07:28:14,552 INFO    MainThread:25294 [redirect.py:_stop():293] _stop closed: stderr
2021-03-15 07:28:14,552 INFO    stderr    :25294 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-03-15 07:28:14,552 INFO    stderr    :25294 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-03-15 07:28:14,552 INFO    MainThread:25294 [redirect.py:_stop():299] _stop joined: stderr
2021-03-15 07:28:14,552 INFO    MainThread:25294 [redirect.py:_stop():301] _stop rd closed: stderr
2021-03-15 07:28:14,552 INFO    MainThread:25294 [redirect.py:uninstall():236] uninstall done
2021-03-15 07:28:15,145 INFO    MainThread:25294 [wandb_run.py:_wait_for_finish():1572] got exit ret: file_counts {
  wandb_count: 1
  media_count: 2
}
pusher_stats {
  uploaded_bytes: 33461
  total_bytes: 36110
}

2021-03-15 07:28:17,150 INFO    MainThread:25294 [wandb_run.py:_wait_for_finish():1572] got exit ret: file_counts {
  wandb_count: 5
  media_count: 2
}
pusher_stats {
  uploaded_bytes: 512983
  total_bytes: 512983
}

2021-03-15 07:28:19,156 INFO    MainThread:25294 [wandb_run.py:_wait_for_finish():1572] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 2
}
pusher_stats {
  uploaded_bytes: 512983
  total_bytes: 512983
}

2021-03-15 07:28:20,660 INFO    MainThread:25294 [wandb_run.py:_show_summary():1727] rendering summary
2021-03-15 07:28:20,661 INFO    MainThread:25294 [wandb_run.py:_show_history():1765] rendering history
2021-03-15 07:28:20,663 INFO    MainThread:25294 [wandb_run.py:_show_files():1794] logging synced files
